k8s/src/main/scala/io/buoyant/k8s/PortMapLogging.scala

```diff
@@ -0,0 +1,35 @@
+package io.buoyant.k8s
+
+import com.twitter.logging.Logger
+
+private[k8s] trait PortMapLogging {
```

Comment:
Would prefer to make this a class with `nsName` and `serviceName` as constructor arguments instead of a trait